spark-shell/src/main/scala/org/apache/mahout/sparkbindings/shell/MahoutSparkILoop.scala

```diff
@@ -157,7 +157,7 @@ class MahoutSparkILoop extends SparkILoop {
          _ __ ___   __ _| |__   ___  _   _| |_
         | '_ ` _ \ / _` | '_ \ / _ \| | | | __|
         | | | | | | (_| | | | | (_) | |_| | |_
-        |_| |_| |_|\__,_|_| |_|\___/ \__,_|\__|  version 0.11.0
+        |_| |_| |_|\__,_|_| |_|\___/ \__,_|\__|  version 0.11.1-SNAPSHOT
```

Comment:
Keep this as 0.11.1, this file is not modified by the release process. Its only the POMs that r updated from 'xxx-SNAPSHOT' to 'xxx' at the time of release. This one is part of the source code and needs to be updated manually. So lets have that as 0.11.1.